python/ray/tests/spark/test_utils.py

```diff
@@ -136,6 +137,11 @@ def test_verify_node_options():
         )
 
 
+def test_convert_dbfs_path_to_local_path():
+    assert _convert_dbfs_path_to_local_path("dbfs:/xx/yy/zz") == "/dbfs/xx/yy/zz"
+    assert _convert_dbfs_path_to_local_path("dbfs:///xx/yy/zz") == "/dbfs/xx/yy/zz"
```

Comment:
It hardcodes "dbfs", but we need to support UC path too. We can support it in following PR.